assets/javascripts/discourse/initializers/qa-edits.js

```diff
@@ -28,7 +28,7 @@ function initPlugin(api) {
   );
 
   api.removePostMenuButton("reply", (attrs) => {
-    return attrs.qa_enabled;
+    return attrs.qa_enabled && attrs.post_number !== 1;
```

Comment:
Any assertions to make?

Reply:
Good point, I've added a test case for this.